code/mob/living/silicon/robot.dm

```diff
@@ -260,6 +260,11 @@
 			src.remove_syndicate("death")
 		src.borg_death_alert()
 		src.eject_brain(fling = TRUE) //EJECT
+		if (length(src.clothes))
+			for (var/A in src.clothes)
+				var/obj/item/cloth_item = src.clothes[A]
+				if (cloth_item)
+					cloth_item.set_loc(src.loc)
```

Comment:
why not just look for obj item as anything in clothes?

Reply:
changed it, and it works on the /kill command, but appears to not work when a cyborg is killed by blunt damage, not sure why. I'll check it out tomorrow